catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/fs/FileSystemProvider.java

```diff
@@ -32,6 +32,8 @@
  */
 public interface FileSystemProvider {
 
+  String GRAVITINO_BYPASS = "gravitino.bypass.";
```

Comment:
Can you please add some comments to this variable, to let users know how to leverage this variable.